pkg/cmd/helm/build/build.go

```diff
@@ -59,6 +66,13 @@ func NewCmdHelmBuild() (*cobra.Command, *Options) {
 	cmd.Flags().StringVarP(&o.ChartsDir, "charts-dir", "c", "charts", "the directory to look for helm charts to release")
 	cmd.Flags().StringVarP(&o.HelmBinary, "binary", "n", "", "specifies the helm binary location to use. If not specified defaults to 'helm' on the $PATH")
 	cmd.Flags().BoolVarP(&o.UseHelmPlugin, "use-helm-plugin", "", false, "uses the jx binary plugin for helm rather than whatever helm is on the $PATH")
+	cmd.Flags().StringVarP(&o.RepositoryUsername, "repo-username", "", "", "the username to access the chart repository. If not specified defaults to the environment variable $JX_REPOSITORY_USERNAME")
+	cmd.Flags().StringVarP(&o.RepositoryPassword, "repo-password", "", "", "the password to access the chart repository. If not specified defaults to the environment variable $JX_REPOSITORY_PASSWORD")
+
+	cmd.Flags().BoolVarP(&o.OCI, "oci", "", false, "using OCI charts")
+	cmd.Flags().StringVarP(&o.RegistryConfigFile, "registry-config", "", "/tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson", "the path to the registry config for OCI login")
```

Comment:
I get the impression from the code that --registry-config or ----repo-password togehter with --repo-username is mandatory together with --oci. This should then be stated in the usage and also validated (typically in the Validate method).

I don't see that JX_REPOSITORY_USERNAME or JX_REPOSITORY_PASSWORD actually are read.